### PR TITLE
RD-3304 Cross Border Support

### PR DESCRIPTION
**Pull request by daniellin215**

### CODE REVIEW

File: core/src/types.js [hunk 1]
```diff
@@ -166,6 +166,7 @@ export type Bank = {
     provider: string,
     country: string,
     identifier: string,
+    supportsTransferDestinationTypes: Array<string>, // A list of Transfer Destination Types, like SEPA, ELIXIR, supported by the bank.
```
Comment (Prateek-Gupta1) [inline core/src/types.js]: supportedTransferDestinationTypes

File: core/src/types.js [hunk 2]
```diff
@@ -289,3 +290,9 @@ export type StandingOrderSubmission = {
     standingOrderBody: Object,
     status: SubmissionStatus,
 };
+
+export type TokenRequestTransferDestinationsCallbackParameters = {
+    supportedTransferDestinationType: Array<string>,
```
Comment (Prateek-Gupta1) [inline core/src/types.js]: supportedTransferDestinationTypes

File: tpp/src/http/AuthHttpClient.js [hunk 3]
```diff
@@ -125,6 +125,26 @@ class AuthHttpClient extends CoreAuthHttpClient {
...
+     * @return {Promise} response to the API call
+     */
+    async setTokenRequestTransferDestinations(tokenRequestId, transferDestinations) {
+        const req = {
+            tokenRequestId,
+            transferDestinations,
+        };
+        const request = {
+            method: 'put',
+            url: `/token-requests/set/transfer-destinations/${tokenRequestId}`,
```
Comment (Prateek-Gupta1) [inline tpp/src/http/AuthHttpClient.js]: Sorry, I modified the Url later and forgot to share the change. Its over here > https://github.com/tokenio/lib-proto/commit/d3f3c1fd316cc33c36f8dcba810ad13daa20afa7

File: core/src/Util.js [hunk 4]
```diff
@@ -205,12 +205,17 @@ export class Util {
         return Util.wrapBuffer(base64UrlToBuffer(key));
     }
 
-    static parseParamsFromUrl(url) {
+    static parseParamsFromUrl = url => {
```
Comment (zixunjian) [inline core/src/Util.js]: keep consistent with other methods in this class

File: core/src/Util.js [hunk 5]
```diff
@@ -205,12 +205,17 @@ export class Util {
...
         const query = url.split('?')[1];
         const result = {};
+        const supportedTypes = [];
         query.split('&').forEach(function(part) {
             const item = part.split('=');
-            result[item[0]] = decodeURIComponent(item[1]);
+            if (item[0] === 'supportedTransferDestinationType') { // for parsing set transfer destinations url callback parameters
```
Comment (zixunjian) [inline core/src/Util.js]: this method is for generic URL param parsing, there shouldn't be this kind of specific logic here. also why do we need this?

**Pull request merged**